src/Components/ProgressiveOnboarding/ProgressiveOnboardingPopover.tsx

```diff
@@ -1,17 +1,21 @@
 import { FC } from "react"
-import { Box, Popover, PopoverProps } from "@artsy/palette"
+import { Box, BoxProps, Popover, PopoverProps } from "@artsy/palette"
 import { Z } from "Apps/Components/constants"
 import { useProgressiveOnboardingTracking } from "Components/ProgressiveOnboarding/useProgressiveOnboardingTracking"
 
 interface ProgressiveOnboardingPopoverProps
   extends Omit<PopoverProps, "children"> {
   name: string
+  boxProps?: BoxProps
 }
 
 export const ProgressiveOnboardingPopover: FC<ProgressiveOnboardingPopoverProps> = ({
   popover,
   children,
   name,
+  boxProps = {
+    width: "fit-content",
+  },
```

Comment:
So — I hate this. I'm going to see if I can fix the underlying problem of why this `fit-content` was introduced in the first place.